examples/02-registered-shopper-auth-token.ts

```diff
@@ -8,74 +8,52 @@
 /**
  * Get authorization token for a registered customer
  * Usage: ts-node examples/02-registered-shopper-auth-token.ts
- * Note: Replace configuration parameters and user credentials before running
  */
 import { ClientConfig, Customer } from "../dist";
```

Comment:
```suggestion
import { ClientConfig, Customer } from "commerce-sdk";
```
If we import from `../dist`, users need to clone the repo and run `npm run build` before the example will run.

As a workaround to get the types working, you may need a package.json in the examples dir.

Reply:
``` json
{
  "name": "commerce-sdk-examples",
  "description": "Example usage of the Commerce SDK",
  "dependencies": {
    "commerce-sdk": "file:.."
  }
}
```